test/e2e/windows/README.md

```diff
@@ -4,7 +4,7 @@
 
 ```bash
 KUBECONFIG=path/to/kubeconfig
-curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/repo-list -o repo_list
+curl https://raw.githubusercontent.com/e2e-win/e2e-win-prow-deployment/master/repo-list -o repo_list # or https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/images/image-repo-list
```

Comment:
Considering that kubernetes-sigs/windows-testing is the official repo, that would make it's image-repo-list file official and up to date. I would just have that file mentioned, no need for both.

Reply:
👍